layouts/_partials/components/analytics/goat-counter.html

```diff
@@ -0,0 +1,17 @@
+{{- with .Site.Params.analytics.goatCounter -}}
+    {{- if .code -}}
+        <script 
+            data-goatcounter="https://{{- .code -}}.goatcounter.com/count"
```

Comment:
```suggestion
            data-goatcounter="https://{{ .code }}.goatcounter.com/count"
```

When using a template inside a line, the `-` are not useful.